Dockerfile

```diff
@@ -0,0 +1,32 @@
+# golang image
+FROM golang:alpine
+
+# install build tools
+RUN apk --no-cache add rrdtool rrdtool-dev git gcc make build-base dropbear-scp
+
+# copy your app into the image
+WORKDIR /app
+ADD . /app
+
+# grab some dependencies
```

Comment:
Could you remove these lines that install dependent packages and use `glide i` instead so that we don't need to update the Dockerfile every time we make a change on required packages? It should look like [the commands in the build process](https://github.com/doublemarket/grafana-rrd-server/blob/3e319cf266c9819047622e3e57196a8e82d75098/.circleci/config.yml#L40-L49). Thanks!